lib/rules/no-raw-text.ts

```diff
@@ -111,7 +111,7 @@ function checkSvelteLiteralOrText(
   const loc = literal.loc!
   context.report({
     loc,
-    message: `raw text '${literal.value}' is used`
+    message: `raw text '${literal.value.trimStart().trimEnd()}' is used`
```

Comment:
I think it can be made simpler.

```suggestion
    message: `raw text '${literal.value.trim()}' is used`
```

Reply:
Hah, of course, for some reason I tried `trim` and it didn't seem to work but I realize now it was because I was hitting the other "raw text is used" string.